includes/class-apple-news.php

```diff
@@ -124,12 +127,28 @@ public static function get_authors( $between = null, $between_last = null, $befo
 		 */
 		$use_cap = apply_filters( 'apple_news_use_coauthors', function_exists( 'coauthors' ), get_the_ID() );
 
+		// Get theme option for byline links. True if set to yes.
+		// Ignore html option if setting meta data.
+		$use_byline_links = $theme->get_value( 'byline_links' ) && 'yes' === $theme->get_value( 'byline_links' ) && 'APPLE_NEWS_DELIMITER' !== $between;
+
 		// Handle CAP authorship.
 		if ( $use_cap ) {
-			return coauthors( $between, $between_last, $before, $after, false );
+			return $use_byline_links
+				? coauthors_posts_links( $between, $between_last, $before, $after, false )
+				: coauthors( $between, $between_last, $before, $after, false );
+		}
+
+		// Get author from post.
+		$post_author_id = intval( $post->post_author );
+		$author         = ucfirst( get_the_author_meta( 'display_name', $post_author_id ) );
+
+		// If we have byline links enabled.
+		if ( $use_byline_links ) {
+			$byline_url = apply_filters( 'apple_news_author_byline_link', get_author_posts_url( $post_author_id ), $post_author_id, get_the_author_meta( 'nicename', $post_author_id ) );
```

Comment:
New filter needs documentation. See other filters for examples.

Reply:
Done https://github.com/alleyinteractive/apple-news/wiki/Filter:-apple_news_author_byline_link